### PR TITLE
feat: Proposal cards in discussions

### LINKED ISSUE #164: Link discussions and proposals in the UI
**User Story**
As a user, I want to be able to navigate between proposals and their parent discussions and vis versa. 

Design still to be completed but some drafts are in the figma

### PR DESCRIPTION
**Pull request by rossneilson**
Closes #164 

Reuses the proposal card in discussions by searching orbis for linked proposals.
If on another chain to than the one the proposal was made on (since discussions are chain agnostic) a message is displayed instead of the data.

Screenshot 2023-02-27 at 15 54 40

Also added a back to discussion button to proposals with a 2 way link and changed the previous button to a home icon 
Screenshot 2023-02-27 at 16 12 28

### CODE REVIEW

**Review by mprasanjith — DISMISSED** [review]

File: apps/davi/src/components/Discussion/Discussion.tsx [hunk 1]
```diff
@@ -75,6 +81,51 @@ function Discussion({
...
+    const _proposals = reset ? [] : [...proposals];
+
+    let { data, error } = await orbis.getPosts(
+      {
+        context: `DAVI-${daoId}-${parentId}-proposal`,
+      },
+      0
+    );
+
+    if (error) console.log(error);
```
Comment (dcrescimbeni) [inline apps/davi/src/components/Discussion/Discussion.tsx:102]: If we're not handling the error, should it be removed?

File: apps/davi/src/components/Discussion/Proposal/Proposal.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { useEffect, useRef } from 'react';
+import { DiscussionMasterPost } from '../Discussion.styled';
+import {
+  // PostBody,
```
Comment (dcrescimbeni) [inline apps/davi/src/components/Discussion/Proposal/Proposal.tsx:4]:
```suggestion
```

File: apps/davi/src/components/Discussion/Proposal/Proposal.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        </PostCreatorAddressBadge>
+        <PostTime>
+          {post?.timestamp && moment.unix(post.timestamp).fromNow()}
+        </PostTime>
+      </PostHeader>
+      <PostWrapper>
+        {proposal?.creator !== '0x0000000000000000000000000000000000000000' ? (
+          <ProposalCardWrapper key={proposalId} proposalId={proposalId} />
+        ) : (
+          <h3>{`Sorry, this proposal sees to be on another chain, try switching to ${post?.content?.data?.chain?.name} to view it`}</h3>
```
Comment (dcrescimbeni) [inline apps/davi/src/components/Discussion/Proposal/Proposal.tsx:57]: We should add a translation.

Also typo on "sees" -> "seems"

File: apps/davi/src/components/Discussion/Proposal/Proposal.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        <PostCreatorName>{getUsername(post?.creator_details)}</PostCreatorName>
+        <PostCreatorAddressBadge>
+          {getBadgeContent(post?.creator_details)}
+        </PostCreatorAddressBadge>
+        <PostTime>
+          {post?.timestamp && moment.unix(post.timestamp).fromNow()}
+        </PostTime>
+      </PostHeader>
+      <PostWrapper>
+        {proposal?.creator !== '0x0000000000000000000000000000000000000000' ? (
```
Comment (dcrescimbeni) [inline apps/davi/src/components/Discussion/Proposal/Proposal.tsx:54]: This could be replaced with `ZERO_ADDRESS`

**Comment by dcrescimbeni** [comment]
I saw this as a change:
image

Was it intentional?

**Review by dcrescimbeni — COMMENTED** [review]
Left some minor comments.

Everything looks good :+1:

**Review by mprasanjith — APPROVED** [review]

**Pull request merged**